Makefile

```diff
@@ -5,11 +5,11 @@
 
 sagefile := .sage/bin/sagefile
 
-.PHONY: $(sagefile)
```

Comment:
I understand why you want to make this change. Some background: We made this a phony target because regenerating the binary was so fast. If we are to remove the phony, we need to reintroduce the file dependencies that this target had, which was `go.sum` and `*.go` files in the `.sage` folder.

Otherwise the binary will never be updated when running locally.

Reply:
You can find the exact stuff here https://github.com/einride/sage/commit/b9faf866ef21fc2b05f81b8e8220300f369b8bd1 :)

Reply:
missing a space after `$(sagefile):` here

Reply:
@odsod @sharkema should we also bring back the logging, which we had before, when regenerating the binary. It was unnecessary when we regenerated each time. But now I think its nice to see when it happens

Reply:
Currently it logs on generation of the binary:
```
[sage] building binary and generating Makefiles...
```

Reply:
Oh, I thought we removed that. No then we are good

Reply:
I thought it was a little bit noisy

Reply:
Yes, but given that we wont recreate it by default, its might be more clear to the user if its now printed?

Reply:
Sounds good to me!

Reply:
I see my confusion, it was removed here indeed; https://github.com/einride/sage/commit/991a4cc2f4b6d11f768862e063f379ebe4644129  though the branch I was working on was depending on an older version